src/transformers/models/speech_to_text_2/modeling_speech_to_text_2.py

```diff
@@ -870,7 +870,7 @@ def forward(
         >>> from datasets import load_dataset
 
         >>> feature_extractor = Wav2Vec2FeatureExtractor()
-        >>> tokenizer = Speech2Text2Tokenizer.from_pretrained(_CHECKPOINT_FOR_DOC)
+        >>> tokenizer = Speech2Text2Tokenizer.from_pretrained("facebook/s2t-wav2vec2-large-en-de")
```

Comment:
```suggestion
        >>> tokenizer = Speech2Text2Tokenizer.from_pretrained({_CHECKPOINT_FOR_DOC})
```

Reply:
Ah I see! Good catch. I think we forgot to change the docstring into a `f-docstring` so that the variables are correctly displayde. We we should put a `f` in the beginning of the docstring so that the variable can be seen. Does that make sense?

Reply:
Thanks, @patrickvonplaten.

So if we do `>>> tokenizer = Speech2Text2Tokenizer.from_pretrained({_CHECKPOINT_FOR_DOC})` in the python model file, than, in the (updated) [documentation page](https://huggingface.co/docs/transformers/model_doc/speech_to_text_2), it will be rendered as `>>> tokenizer = Speech2Text2Tokenizer.from_pretrained("facebook/s2t-wav2vec2-large-en-de")`?